web/impact/impact/urls.py

```diff
@@ -146,10 +147,25 @@
 urls = [
     url(r"^api/v0/", include(v0_urlpatterns)),
     url(r"^api/v1/", include(v1_urlpatterns)),
-    url(r'^api/(?P<app>\w+)/(?P<model>\w+)/$',
+    url(
+        r'^api/(?P<app>\w+)/(?P<model>[a-z_]+)/'
+        r'(?P<related_model>[a-z_]+)/$',
+        GeneralViewSet.as_view({'get': 'list', 'post': 'create'}),
+        name='related-object-list'),
+    url(r'^api/(?P<app>\w+)/(?P<model>[a-z_]+)/'
+        r'(?P<related_model>[a-z_]+)/'
+        r'(?P<pk>[0-9]+)/$',
+        GeneralViewSet.as_view({
+            'get': 'retrieve',
+            'put': 'update',
+            'patch': 'partial_update',
+            'delete': 'destroy'
+        }),
+        name='related-object-detail'),
+    url(r'^api/(?P<app>\w+)/(?P<model>[a-z_]+)/$',
```

Comment:
maybe the `(?P<app>\w+)` part should also be lower-case only? not sure.

Reply:
This doesn't seem to matter.  Leaving as is.